extra_tests/snippets/stdlib_os.py

```diff
@@ -507,3 +507,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     for arg in [None, 1, 1.0, TabError]:
         assert_raises(TypeError, os.system, arg)
+
+# Testing for os.pathconf_names
+if "win" not in sys.platform:
```

Comment:
```suggestion
if not sys.platform.startswith("win"):
```
"darwin" also contains "win", which means macOS.